### ISSUE TITLE
Tensorflow and Time imports

### ISSUE THREAD

**Issue by AlanTJohnstone** [AndreasMadsen/python-lrcurve#9]
1 although I am only using it with Pytorch it tries to import tensorflow which I do not currently have installed - perhaps make it optional.
2 the time module is used but not imported

**Comment by DJCordhose** [comment]
I can double down on 2: 

```
/usr/local/lib/python3.7/dist-packages/lrcurve/plot_learning_curve.py in reconfigure(self, height, width, mappings, line_config, facet_config, xaxis_config)
    181         # if the iframe is not properly initialized yet. Unfortunately, I can't find a way
    182         # to probe if the initialization is complete. Instead, add a 1 second delay.
--> 183         time.sleep(1)
    184         self._update_element.update(disp)
    185 

NameError: name 'time' is not defined
```

**Comment by AndreasMadsen** [comment]
Sorry about that. Unfortunately, this module is really a pain to test automatically, as it depends on a live jupyter-lab instance running in a browser.

Both issues should be fixed now in 2.2.1.

**Comment by AlanTJohnstone** [comment]
Thank you.  A simple but extremely useful bit of software.
I liked the way you dealt with the  need for tensor flow issue.

Alan
